androidlibrary_lib/src/androidTest/java/org/opendatakit/logic/PropertiesNonPrivilegedTest.java

```diff
@@ -64,7 +63,7 @@ public void setUp() throws Exception {
     @Test
     public void testSimpleProperties() {
 
-        Context context = InstrumentationRegistry.getContext();
+        Context context = androidx.test.platform.app.InstrumentationRegistry.getInstrumentation().getTargetContext();
```

Comment:
You should add a new import for "androidx.test.platform.app" that replaces the old import instead of writing it out several times. Imports are used to not have to type out the full class package every time.

Reply:
> You should add a new import for "androidx.test.platform.app" that replaces the old import instead of writing it out several times. Imports are used to not have to type out the full class package every time.

Hi @wbrunette I have pushed the changes as requested for the new imports